### PR TITLE
Feature update btms

### PR DESCRIPTION
**Pull request by ssankalp26**
1. Added a series and parallel bus configuration which the user can specify, and the power split is determined accordingly.
2. Incorporated the addition of the power to the mission solver at the end of each time step
3. Added the cooling drag to the mission solver which will account for it by an increase in thrust

### CODE REVIEW

File: RCAIDE/Library/Components/Propulsors/Electric_Rotor.py [hunk 1]
```diff
@@ -33,7 +33,8 @@ def __defaults__(self):
         self.tag                          = 'electric_rotor'    
         self.motor                        = None
         self.rotor                        = None 
-        self.electronic_speed_controller  = None   
+        self.electronic_speed_controller  = None
+        self.active_busses                = None
```
Comment (ssankalp26) [inline RCAIDE/Library/Components/Propulsors/Electric_Rotor.py:37]: added this back here based on our discussions

File: RCAIDE/Library/Methods/Aerodynamics/Common/Drag/cooling_drag.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    pressure                   = conditions.freestream.pressure
+    reference_area             = state.analyses.aerodynamics.vehicle.reference_area
+    
+ 
+    cd_cooling  = np.zeros_like(density)
+    
+    for network in geometry.networks:
+        for coolant_line in  network.coolant_lines:
+            for tag, item in  coolant_line.items():
+                if tag == 'heat_exchangers':
```
Comment (ssankalp26) [inline RCAIDE/Library/Methods/Aerodynamics/Common/Drag/cooling_drag.py:52]: Will only go through if coolant_line exists else cooling drag will be zero

File: Regressions/Tests/network_electric/electric_btms_test.py [hunk 3]
```diff
@@ -27,9 +27,10 @@
 # ----------------------------------------------------------------------------------------------------------------------  
 def main():           
          
-    battery_types = ['lithium_ion_lfp', 'lithium_ion_nmc']
+    battery_types = ['lithium_ion_nmc', 'lithium_ion_lfp']
     btms_types    = ['Liquid_Cooled_Wavy_Channel', 'Air_Cooled', None] 
-    CL_true       = [0.63136618] 
+    CL_true       = [[0.6044019730641954,0.604775958081217,0.604775958081217],
+                     [0.6044019723858562, .6047759580812461, 0.6047759580812461]]
```
Comment (ssankalp26) [inline Regressions/Tests/network_electric/electric_btms_test.py:33]: change in cl values due to addition of cooling drag, deviation from before is minor (+0.01)

File: RCAIDE/Library/Components/Systems/Avionics.py [hunk 4]
```diff
@@ -42,7 +42,7 @@ def __defaults__(self):
         N/A
         """                 
         self.power_draw = 0.0
-        self.tag        = 'avionics'
+        self.tag        = 'Avionics'
```
Comment (mclarke2) [inline RCAIDE/Library/Components/Systems/Avionics.py:45]: lower case

**Review by mclarke2 — APPROVED** [review]

File: RCAIDE/Library/Components/Systems/Avionics.py (hunk 4 above)
Comment (mclarke2) [reply RCAIDE/Library/Components/Systems/Avionics.py:45]: I will fix this in my code

**Pull request merged**